perl6intro.adoc

```diff
@@ -1400,19 +1401,19 @@ say "1.2 squared is equal to " ~ squared(1.2);
 NOTE: For more info on subroutines and functions, see https://docs.perl6.org/language/functions
 
 == Functional Programming
-In this chapter we will take a look at some of the functionality that facilitates Functional Programming.
+In this chapter we will take a look at some of the features that facilitate Functional Programming.
 
 === Functions are first-class citizens
 Functions/subroutines are first-class citizens:
 
-* They can be passed as an argument
+* They can be passed as arguments
 
-* They can be returned from another function
+* They can be returned from other functions
 
-* They can be assigned to a variable
+* They can be assigned to variables
 
-A great example to demonstrate this concept is the `map` function. +
-`map` is a *higher order function*, it accepts another function as an argument.
+A great example is the `map` function. +
+`map` is a *higher order function*, it cam accept another function as an argument.
```

Comment:
`map` is a *higher order function*, it can accept another function as an argument.